docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml

```diff
@@ -1040,6 +1041,25 @@
           on appropriate nodes.</para>
         </sect4>
       </sect3>
+
+      <sect3 id="ENV_values_yaml">
+        <title>Environment Values</title>
+
+        <para>You can define Environment Variables in a YAML file. The
+        environment values are defined under the <emphasis>#env:</emphasis>
+        portion of the delivered values.yaml as illustrated below.</para>
+
+        <programlisting>#env: 
```

Comment:
should be

```
global:
  env:
  - name: SMTPserver 
  - value: mysmtpserver
```

You can also specify environment variables for each individual component.  Has that been documented?

Reply:
no. this is the first mention of env
will see to add it here